src/utils/boardImportUtils.ts

```diff
@@ -103,4 +103,12 @@ const uploadAction = (
   return promise;
 };
 
-export { openFromFileSystem, importAssetsToS3 };
+const openAssetsFromFileSystem = async () => {
+  return await fileOpen({
+    description: "Image",
+    extensions: [".png", ".svg", ".jpg", ".jpeg", ".gif"],
```

Comment:
Maybe it would be better to create some kind of mapper from mime-types to extensions?
Allowed mime-types are set in the envs.TLDRAW__ASSETS_ALLOWED_MIME_TYPES_LIST env var.
It will probably be something like this:
`const filemimetypes = {
  "image/jpeg": [".jpg", ".jpeg"],
  "image/png": [".png"],
  ...
};`
No need for npm package since there are not that many types that tldraw supports.